mailing_list/management/commands/activate_all_lists.py

```diff
@@ -45,10 +46,18 @@ def handle(self, *args, **options):
                 failures[course['id']] = str(e)
                 continue
 
+            # the "is it the primary list" logic varies depending on whether or
+            # not the list is crosslisted.  figure out which logic to use
+            # before bucketing the lists.
+            if any([l['is_course_list'] for l in lists]):
+                is_primary = itemgetter('is_course_list')
+            else:
+                is_primary = itemgetter('is_primary')
```

Comment:
I had to lookup the use of dynamically creating functions using the itemgetter :-). (Have mostly used it in sorting). If the lists has a combination of items containing both 'is_course_list' and ''is_primary' - will  the is_primary function work for both?